### PR TITLE
Added the whole section of Hacktober Fest 2022

### PR DESCRIPTION
**Pull request by guptamanu409**
#Added Intro
#Attached all the links
#Made alot of changes which you will really like
Please review and merge my pull request under Hacktober Fest 2022 Please also add me in Contributors

### CODE REVIEW

**Comment by Susmita-Dey** [comment]
@guptamanu409 Issue no?

**Comment by guptamanu409** [comment]
> @guptamanu409 Issue no?

I have updated your Readme.... Added the whole section of Hacktober Fest 2022.... Please review and merge mam

File: README.md [hunk 1]
```diff
@@ -20,9 +20,83 @@ Link to the extension [source code](https://github.com/Susmita-Dey/Moody).
...
+- If it fulfills criteria, I will definitely merge it. 
+- Hurry up guys...!! You are only few steps away to complete your Hacktoberfest challenge. Make 4 pull requests and complete your hacktoberfest challenge and get a chance to win T-shirt. 
+- It is for only 1 month.
+
+# 📚 Contribution rules
+- Do NOT remove other content.
+- Add your name to the <b>CONTRIBUTORS.md</b> file
+- Add me as reviewer
+- Try to keep pull requests small to minimize merge conflicts.</li>
+
```
Comment (Susmita-Dey) [inline README.md:48]: This lines are not okay. Change it.

File: README.md [hunk 2]
```diff
@@ -20,9 +20,83 @@ Link to the extension [source code](https://github.com/Susmita-Dey/Moody).
...
+git push origin my-new-branch
+```
+
+- Create a new pull request from your forked repository
+
+# ❓ FAQs (Frequently Asked Questions)
+
+- Who all can contribute?
+  - Anyone with a github account and who is signed up for
+[hacktoberfest](https://hacktoberfest.digitalocean.com/) 2022 :)
```
Comment (Susmita-Dey) [inline README.md:87]: Apart from hacktoberfest, anyone can contribute. This repository is open for all.

File: README.md (hunk 1 above)
Comment (guptamanu409) [reply README.md:48]: > This lines are not okay. Change it.

So what should i write could you please specify anything or should i remove it

**Comment by Susmita-Dey** [comment]
@guptamanu409 You should have made issue first, then get assigned and then make this PR

File: README.md (hunk 2 above)
Comment (guptamanu409) [reply README.md:87]: > Apart from hacktoberfest, anyone can contribute. This repository is open for all.

Okay lemme change it

**Comment by guptamanu409** [comment]
> @guptamanu409 You should have made issue first, then get assigned and then make this PR

I'm new to this.... This hacktober fest is my first time contributing

**Comment by Susmita-Dey** [comment]
@guptamanu409 I understand. First go and create an issue with the same name as of this PR

**Comment by guptamanu409** [comment]
> @guptamanu409 I understand. First go and create an issue with the same name as of this PR

Mam Hacktober Fest Intro video.... It was not mentioned that we were supposed to first raise issue. 
We just have to fork it then make the changes.

**Review by Susmita-Dey — APPROVED** [review]
LGTM.

**Pull request merged**